### DISCUSSION TITLE
Change start splash NV14

### DISCUSSION THREAD

**Discussion by Ktr128** [EdgeTX/edgetx#1551] [Ideas]
I propose to change the start splash for NV14, for example, to the one shown in the photo:
<a href="https://ibb.co/zFhtpX5"><img src="https://i.ibb.co/zFhtpX5/splash.png" alt="splash" border="0"></a>
<a href="https://ibb.co/wL4hWv7"><img src="https://i.ibb.co/wL4hWv7/Screenshot-20220128-210607-Gallery.jpg" alt="Screenshot-20220128-210607-Gallery" border="0"></a>

**Comment by Ktr128** [comment]
Splash Glider Edition 

![background2-min](https://user-images.githubusercontent.com/98200485/160281418-cd818860-acbf-46ad-8b35-4a09d63c7ab8.png)

**Comment by pfeerick** [comment]
I'm not sure if this will happen, as EdgeTX tries to be manufacturer agnostic. As such, all radios should have only a standard splash screen by default. If you, as the end user however, want a custom splash screen, that's all good. That's what the `/IMAGES/splash.png` path is for. However, in addition to the [themes directory/repo](https://github.com/EdgeTX/themes)... perhaps we should also have somewhere to collect splash screens for download?

**Comment by Ktr128** [comment]
If there is a users need for additional storage space for splash.png, let's create it. I will move my splash.png there. I will be glad if someone needs them.
For myself, I compile the firmware with the splash I need ...

**Reply by Ktr128** [reply]
P.S. That's a little forward of me.
I did not know that at startup the images folder is monitored in search of custom splash.png :)

**Reply by pfeerick** [reply]
Yes, colorlcd is nicer than B&W... you don't need the compile the splash screen in ;)
